config/monitoring.sensor_config.ocha_google_tag.yml

```diff
@@ -1,11 +1,9 @@
-uuid: 387db73c-d76b-4cf8-aec2-28e9c8531804
+uuid: null
```

Comment:
Think this config should be removed

Reply:
It will be created again by ocha_monitoring

Reply:
See https://github.com/UN-OCHA/ocha_monitoring/pull/4